vue-frontend/src/App.vue

```diff
@@ -1,26 +1,17 @@
 <script setup>
 import LeafletMap from './components/map/LeafletMap.vue'
-import Slider from "@/components/slider/Slider.vue";
-import {ref} from "vue";
 
-const time = ref(new Date().getTime())
 </script>
 
 <template>
 
   <main>
 
     <div>
-      <!--      <div style="float:left; width: auto">-->
-      <!--            <KnowledgeGraph/>-->
-      <!--      </div>-->
       <div>
         <LeafletMap></LeafletMap>
```

Comment:
Maybe to find a more specific, less technology specific, name for this component?